server/deps.edn

```diff
@@ -23,6 +23,7 @@
         incanter/incanter {:mvn/version "1.9.3"
                            :exclusions [bouncycastle/bctsp-jdk14
                                         postgresql/postgresql]}
+        com.taoensso/nippy {:mvn/version "3.4.2"}
```

Comment:
We use nippy to serialize the `data` field, since it can be anything and nippy is good at serializing any clojure data structure efficiently: https://github.com/taoensso/nippy